phoenix-core/src/main/java/org/apache/phoenix/parse/ColumnParseNode.java

```diff
@@ -40,6 +41,13 @@ public ColumnParseNode(TableName tableName, String name, String alias) {
         this.tableName = tableName;
         fullName = tableName == null ? getName() : tableName.toString() + QueryConstants.NAME_SEPARATOR + getName();
     }
+    
+    public ColumnParseNode(TableName tableName, String name, String alias, boolean caseSensitive) {
```

Comment:
This is needed so that expressions that refer to case senstive columns work correctly.